src/main/java/org/broadinstitute/hellbender/tools/spark/sv/evidence/BreakpointEvidence.java

```diff
@@ -86,6 +91,32 @@ public BreakpointEvidence read( final Kryo kryo, final Input input, final Class<
         }
     }
 
+    @DefaultSerializer(ExternalEvidence.Serializer.class)
+    public final static class ExternalEvidence extends BreakpointEvidence {
+        public ExternalEvidence( final SVInterval interval, final int weight ) {
```

Comment:
For the other evidence types, we're adding the uncertainty to their intervals here in their constructors in `BreakpointEvidence`, whereas you're adding the uncertainty prior to creating the object. Is that because we've parameterized this uncertainty but not the other types of uncertainty (which are still static constants)? If there's not a technical reason to do so it might be nice to have one place to look for this sort of logic. Not a really big deal, though.

Reply:
Yes, good idea.  I'll reuse that code.